packages/brand-moment/package.json

```diff
@@ -41,6 +41,8 @@
     "classnames": "^2.3.1"
   },
   "devDependencies": {
+    "@cultureamp/frontend-core": "^2.1.1",
+    "@cultureamp/unified-navigation-ui": "^2.0.1",
```

Comment:
Really glad I set up the private packages now :D

Reply:
Unsure what you mean by this? 😅